pom.xml

```diff
@@ -79,15 +78,16 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.222.x</artifactId>
-        <version>21</version>
+        <artifactId>bom-2.249.x</artifactId>
+        <!-- TODO weekly -->
+        <version>966.v3857b7c82032</version>
```

Comment:
@jglick - I lifted this from some other plugin I saw you update, but I'm not sure if this is still the "right" thing to do for the BOM. Got any pointers?

Reply:
Use `bom-weekly`.